index.html

```diff
@@ -38,8 +39,42 @@ <h2>Selected Publications</h2>
     </ul>
   </section>
 
-<section id="contact"></section>
-<h2>Contact</h2>
+<section id="contact">
+  <h2>Contact</h2>
+    <h3>Phone:</h3>
+      <p>01234/56789 </p>
+    <h3>Mail: </h3>
+      <p>m.schuetz@outlook.de</p>
+</section>
+
+<section>
+<h2>I really wanted some clickable icons here, but I'll settle for images for now</h2>
+  <figure>
+  <a href="https://www.linkedin.com/in/dr-magdalena-sch%C3%BCtz-625a781a6">
+    <img src="https://source.unsplash.com/random/600x300"
+    width=70" height="70" alt="">
+ </a>
+</figure> 
+<figcaption>LinkedIn again</figcaption>
```

Comment:
I am not sure about this...[MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption) says figcaption must be first or last child, i believe, this means wrapped inside `<figure></figure>`